### DISCUSSION TITLE
buffalo_l and antelopev2

### DISCUSSION THREAD

**Discussion by ZSY4** [immich-app/immich#7428] [Q&A]
1、buffalo_l and antelopev2
Excuse me, which of the two models is better
2、MIN DETECTION SCORE，MAX RECOGNITION DISTANCE，MIN RECOGNIZED FACES
I also want to know how these three parameters should be set, which values are suitable for which model

I am currently using antelopev2，MIN DETECTION SCORE:0.6，MAX RECOGNITION DISTANCE:0.5，MIN RECOGNIZED FACES:4

I don't know if antelopev2 is the best model, so there is question 1

**Comment by aviv926** [comment]
https://github.com/immich-app/immich/discussions/5028
